apps/cowswap-frontend/src/modules/permit/hooks/useIsTokenPermittable.ts

```diff
@@ -21,17 +24,22 @@ import { checkIsTokenPermittable } from '../utils/checkIsTokenPermittable'
  * When it is not, returned type is `false`
  * When it is unknown, returned type is `undefined`
  *
- * @param token
  */
-export function useIsTokenPermittable(token: Nullish<Currency>): IsTokenPermittableResult {
+export function useIsTokenPermittable(
+  token: Nullish<Currency>,
+  tradeType: Nullish<TradeType>
```

Comment:
You probably can just use `useTradeTypeInfo()`

Reply:
I didn't want to use that because it has url route info and I feel like it doesn't make sense to bring that dependency onto this hook/module